src/Microsoft.PowerShell.Commands.Management/commands/management/Computer.cs

```diff
@@ -223,6 +261,74 @@ public class RestartComputerCommand : PSCmdlet, IDisposable
         [Alias("f")]
         public SwitchParameter Force { get; set; }
 
+        /// <summary>
+        /// Provide a comment for rebooting the computer.
+        /// </summary>
+        [Parameter(ParameterSetName = DefaultParameterSet)]
+        public string Comment { get; set; } = "";
```

Comment:
Please use `string.Empty`.

Style issue: all public comments for properties with `{ get; set; }` should start with "Gets or sets"; (line 265)